nifi-nar-bundles/nifi-redis-bundle/nifi-redis-extensions/src/test/java/org/apache/nifi/redis/service/TestRedisConnectionPoolService.java

```diff
@@ -39,6 +61,69 @@ public void setup() throws InitializationException {
         testRunner.addControllerService("redis-service", redisService);
     }
 
+    private void enableSslContextService(final SSLContext sslContext) throws InitializationException {
```

Comment:
The `SSLContext` parameter can be removed since it is already a private static variable.
```suggestion
    private void enableSslContextService() throws InitializationException {
```